tests/profiling/settings.js

```diff
@@ -1,11 +1,13 @@
 const {getAssets} = require('./dao.js');
+//const redisStore = require('ha-redis-adapter');
```

Comment:
Should we add as dev dependency?

Reply:
It could be useful for testing, I'll add it.